tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Handler.java

```diff
@@ -68,8 +73,11 @@ public void initialize() {
 
         // TODO: Initialize the thing. If done set status to ONLINE to indicate proper working.
         // Long running initialization should be done asynchronously in background.
-        updateStatus(ThingStatus.ONLINE);
 
+        // logger.debug("Start initializing!");
```

Comment:
IMHO this will lead developers to add a lot of debug logs that are not really necessary.
But it is okay as long as something is in work and should be removed before merging it.

Shouldn't the "start initializing" moved to the first line of that method?